lib.sh

```diff
@@ -430,11 +430,41 @@ else
 fi
 }
 
-# Let's Encrypt for subdomains
-le_subdomain() {
+#generate certs and auto-configure
+# https://certbot.eff.org/docs/using.html#certbot-command-line-options
+generate_cert() {
+if [[ "no" == $(ask_yes_or_no "Do you need to add the upgrade-insecure-requests(UIR) and Strict-Transport-Security(HSTS) header to every HTTP response?") ]]
```

Comment:
Why ask? 99% of all the users doesn't even know what that meaning means. 

It's added due to security. The more secure the better.

Reply:
The onlyoffice and collabora have no need of HSTS and UIR.

Reply:
Then make it an if statement instead. 

```
if [ -z $SUBDOMAIN ]
then
    add HSTS and UIR
fi
```